### PR TITLE
feat: allow generating accessible pdf with `printToPDF`

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Refs https://bugs.chromium.org/p/chromium/issues/detail?id=1469961
Refs https://github.com/electron/electron/pull/39375.

`Page.printToPDF` now allows for experimentally generating tagged (accessible) PDFs. We needed to add the property to our implementation in the latest Chromium roll in order to fix compilation errors, but we should surface this new property to developers with a note about its experimental status.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] relevant documentation, tutorials, templates and examples are changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/main/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/main/README.md#examples).

#### Release Notes

Notes: Added a `generateTaggedPDF` option to `webContents.printToPDF` to allow generating  tagged (accessible) PDFs.

### CODE REVIEW

File: spec/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "mocha": "^10.0.0",
     "mocha-junit-reporter": "^1.18.0",
     "mocha-multi-reporters": "^1.1.7",
-    "pdfjs-dist": "^2.2.228",
```
Comment (codebytere) [inline spec/package.json:28]: This is needed to pull in `getMarkInfo()`

File: docs/api/web-contents.md [hunk 2]
```diff
@@ -1626,6 +1626,7 @@ win.webContents.print(options, (success, errorType) => {
   * `headerTemplate` string (optional) - HTML template for the print header. Should be valid HTML markup with following classes used to inject printing values into them: `date` (formatted print date), `title` (document title), `url` (document location), `pageNumber` (current page number) and `totalPages` (total pages in the document). For example, `<span class=title></span>` would generate span containing the title.
   * `footerTemplate` string (optional) - HTML template for the print footer. Should use the same format as the `headerTemplate`.
   * `preferCSSPageSize` boolean (optional) - Whether or not to prefer page size as defined by css. Defaults to false, in which case the content will be scaled to fit the paper size.
+  * `generateTaggedPDF` boolean (optional) - Whether or not to generate a tagged (accessible) PDF. Defaults to false. This property is **experimental** and the generated PDF may not adhere fully to PDF/UA and WCAG standards.
```
Comment (dsanders11) [inline docs/api/web-contents.md:1629]: Should we mark these with the `_Experimental_` tag so it gets more officially identified as such?

**Review by zcbenz — APPROVED** [review]
API LGTM

File: docs/api/web-contents.md (hunk 2 above)
Comment (codebytere) [reply docs/api/web-contents.md:1629]: @dsanders11 done!

**Review by ckerr — APPROVED** [review]
API LGTM

**Pull request merged**